src/Textarea.tsx

```diff
@@ -4,15 +4,15 @@ import {TextInputBaseWrapper} from './_TextInputWrapper'
 import {FormValidationStatus} from './utils/types/FormValidationStatus'
 import sx, {SxProp} from './sx'
 
+export const DEFAULT_TEXTAREA_ROWS = 7
+export const DEFAULT_TEXTAREA_COLS = 30
+export const DEFAULT_TEXTAREA_RESIZE = 'both'
+
 export type TextareaProps = {
   /**
    * Apply inactive visual appearance to the Textarea
    */
   disabled?: boolean
-  /**
-   * Indicates whether the Textarea is a required form field
-   */
-  required?: boolean
```

Comment:
This was taken out because we already get it in `TextareaHTMLAttributes<HTMLTextAreaElement>`